### DISCUSSION TITLE
Run the playwright test several times

### DISCUSSION THREAD

**Discussion by yogrik** [artilleryio/artillery#2013] [Artillery]
I have a test written on JavaScript. 
I want to run it several times to increase the precision of the measurement.
At the same time, I don't need the concurrency, so only one user must work with the site simultaneously.
I saw that the HTTP engine has a loop for the same purpose, but the playwright engine does not.
I understand that I can write a loop inside the test, but it doesn't look fair.
Also, I can write several sequenced phrases divided by sleep. It looks better, but it is only convenient for a few iterations, and I need to calculate some artificial timeout, which will slow down the test executions and be hard to maintain.

Are there other approaches?

**Comment by bernardobridge** [comment]
Hello @yogrik 

You should be able to use `maxVusers` (https://www.artillery.io/docs/reference/test-script#constant-arrival-rate) in combination with an `arrivalRate` of 1 to accomplish that, as that limits the number of concurrent users:

```yaml
- duration: 60
  arrivalRate: 1
  maxVusers: 1
```

This should drop VU's from executing if there are still others running, giving you time for each of your scenarios to finish before a new one kicks in.

**Comment by yogrik** [comment]
Wow, I didn't notice the elephant. Thank you!
